composer.json

```diff
@@ -4,6 +4,9 @@
         "ext-gd": "0",
         "ext-curl": "0"
     },
+    "suggest": {
+        "ext-imagick": "to use the Imagick implementation",
```

Comment:
only in watermarkrand - other transformers not require imagick, and working without it.
firstly was in required but lukasz suggested to move it to section suggest